recipes/g2aero/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "G2Aero" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com//NREL/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e5673caaf7af576050ea79659ee3b1e2d641e77f16918633e2a9bfaddce51294
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - pyyaml
+    - scipy
+    - setuptools
+
+test:
+  imports:
+    - g2aero
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/NREL/G2Aero
+  summary: 'G2Aero: Separable shape tensors for aerodynamic applications'
+  description: |
+    G2Aero is a flexible and practical tool for design and deformation of 
+    2D airfoils and 3D blades using data-driven approaches. G2Aero 
+    utilizes the geometry of matrix manifolds—specifically the Grassmannian
+    to build a novel framework for representing physics-based separable 
+    deformations of shapes. G2Aero offers the flexibility to generate 
+    perturbations in a customizable way over any portion of the blade. 
+    The G2Aero framework utilizes data-driven methods based on a curated 
+    database of physically relevant airfoils.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
```

Comment:
@jefalon You're not listing yourself as a maintainer, is that correct?

Reply:
Correct, I'm help Olga and Zach get this package onto conda-forge since I have experience doing it in the past.